lms/static/scripts/frontend_apps/config.ts

```diff
@@ -220,6 +221,10 @@ export type Product = {
   api: ProductAPI;
 };
 
+export type WeekDay = 1 | 2 | 3 | 4 | 5 | 6 | 7;
```

Comment:
The original specification stated days would be identified by their index, but I just realized this has changed and now they are identified like this:

```json
{
  "instructor_email_digests.days.mon": true,
  "instructor_email_digests.days.tue": true,
  "instructor_email_digests.days.wed": true,
  "instructor_email_digests.days.thu": true,
  "instructor_email_digests.days.fri": true,
  "instructor_email_digests.days.sat": true,
  "instructor_email_digests.days.sun": true,
}
```

I'll do the changes.